services/src/main/java/org/jboss/windup/web/services/rest/MigrationProjectEndpointImpl.java

```diff
@@ -76,6 +77,9 @@ public MigrationProject getMigrationProject(Long id)
     @Override
     public MigrationProject createMigrationProject(MigrationProject migrationProject)
     {
+        if (null != getProjectIdByName(migrationProject.getTitle()))
```

Comment:
And @jsight  Actually it adds even in Java, consider:
Boolean myBoolean = true;
if( myBoolean = null )
    -> NPE in runtime